src/jimgw/run/library/IMRPhenomPv2_standard_cbc.py

```diff
@@ -87,31 +72,50 @@ def __init__(
         self.dL_range = dL_range
         self.t_c_range = t_c_range
         self.phase_c_range = phase_c_range
-        self.psi_prior = psi_prior
-        self.ra_prior = ra_prior
+        self.psi_range = psi_range
+        self.ra_range = ra_range
+        self.dec_range = dec_range
 
-        self.likelihood = self.initialize_likelihood()
+    def initialize_jim_objects(self):
```

Comment:
By bringing this out of the init and put it in the run_manager class, this allows dry run capability.